apps/docs/src/content/reference/extras/instanced-spritesheet.mdx

```diff
@@ -0,0 +1,230 @@
+---
+{
+  order: 3,
+  category: '@threlte/extras',
+  sourcePath: 'packages/extras/src/lib/components/InstancedSprite/Spritesheet.svelte',
+  name: <Spritesheet>,
+  type: 'component',
+  componentSignature:
+    {
+      props:
+        [
+					{
+            name: 'dataUrl',
+            type: 'string',
+            required: false,
+            description: 'Url of a spritesheet json with metadata.'
+          },
+          {
+            name: 'textureUrl',
+            type: 'string',
+            required: false,
+            description: 'Url of a spritesheet image. This will get overwritten if you declare spritesheet meta with help of <File/> and <Animation/> child components.'
+          },
+					{
```

Comment:
Why is this boolean needed to load json metadata? Is it not enough to have dataUrl?

Reply:
This is strictly for aseprite jsons. It then gets (roughly-wip) translated to the native spritesheet format for my library looks like this.

```ts
type SpritesheetFormat = {
  frames: [x: number, y: number, w: number, h: number][];
  animations: Record<string, [frameId: number, duration: number][]>;
  sheetSize: [w: number, h: number][];
  animationLengths: number[];
};
```

You could load a json with this format too. Maybe in the future if there's demand it could be like:
`dataFormat: 'aseprite' | 'three-kit' | 'anotherProgram' | 'anotherProgram'`